pyperformance/requirements.txt

```diff
@@ -24,6 +24,10 @@ markupsafe==1.1.1
     # via mako
 mpmath==1.2.1
     # via sympy
+mypy-extensions==0.4.3
+    # via mypy
+mypy==0.812
```

Comment:
This is probably going to pick up the extension if the wheel is available and the pure python if not, which may give different numbers in different versions. Am I mistaken? Maybe I am missing something.

Also, is your intent to test the extension?